x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -418,16 +410,14 @@ export const getMonitoringColumns = (columnsProps: GetColumnsProps): TableColumn
     {
       field: 'current_status.status',
       name: i18n.COLUMN_LAST_RESPONSE,
-      render: (value: RuleStatus['current_status']['status'] | undefined) => (
-        <RuleExecutionStatusBadge status={value} />
-      ),
+      render: (value: Rule['status'] | undefined) => <RuleExecutionStatusBadge status={value} />,
       width: '12%',
       truncateText: true,
     },
     {
       field: 'current_status.status_date',
       name: i18n.COLUMN_LAST_COMPLETE_RUN,
-      render: (value: RuleStatus['current_status']['status_date'] | undefined) => {
+      render: (value: Rule['status_date'] | undefined) => {
```

Comment:
Should `field: 'current_status.status_date'` be updated to `field: 'status_date'`?

I'm testing it and **Last response** (`status`) and **Last run** (`status_date`) seem to work in the Rule Management table:

![](https://puu.sh/IuiWA/fd999f5196.png)

but are displayed as empty in the Monitoring table:

![](https://puu.sh/IuiZX/846e28bd36.png)

Reply:
Yes, `current_status.status_date` should be `status_date `. Missed that piece after rebase.
Thanks for checking that!